src/index.css

```diff
@@ -1,13 +1,3 @@
-body {
```

Comment:
Did we install a normalize file yet? I can't remember, but wanted to mention it

Reply:
I installed normalize ✅

Reply:
Great I appreciate all of this.